contao/classes/Frontend.php

```diff
@@ -442,13 +442,9 @@ public static function addToUrl($strRequest, $blnIgnoreParams=false, $arrUnset=a
 		if (\Config::get('addLanguageToUrl'))
```

Comment:
Maybe a stupid question but shouldn't we use the `prepend_locale` here as well? Or am I missing something now? :D